app/adapters/user.js

```diff
@@ -0,0 +1,19 @@
+import V3Adapter from 'travis/adapters/v3';
+
+export default V3Adapter.extend({
+  buildURL: function (modelName, id, snapshot, requestType, query) {
```

Comment:
That looks good to me. I like using `queryRecord` for that kind of stuff and I think it's fine to handle only that since API doesn't have anything else anyway.